kuka_eki_hw_interface/include/kuka_eki_hw_interface/kuka_eki_hw_interface.hpp

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2018, 3M
```

Comment:
Please don't rename this file for now. We will work on this when we optimize EKI interface later